src/Compilers/CSharp/Test/Syntax/Parsing/ParserErrorMessageTests.cs

```diff
@@ -4816,10 +4816,7 @@ unsafe public class Test
             CreateCompilation(test, options: TestOptions.UnsafeDebugDll).VerifyDiagnostics(
                 // (4,14): error CS0518: Predefined type 'System.Span`1' is not defined or imported
```

Comment:
>// (4,14): error CS0518: Predefined type 'System.Span`1' is not defined or imported [](start = 16, length = 83)

Would this line use System.Span if it was available?

Reply:
It uses it even though it is not available.

---
In reply to: [359107468](https://github.com/dotnet/roslyn/pull/40457#discussion_r359107468) [](ancestors = 359107468)